README.md

```diff
@@ -205,3 +205,25 @@ DB_PASS=secret123
 DB_NAME=main
 DB_WRITE=false
 ```
+
+## Integration tests
+
+Integration tests are defined on `test/integration_test.go`. `make integration-test` Runs this script on the current kubernetes namespace. Given that the test image with your changes is available on that namespace. If you don't have access to a kubernetes namespace you can run tests locally using a Kind (kubernetes in docker) cluster by running `make integration-test-kind`.
```

Comment:
The phrase `Given that the test image with your changes is available on that namespace` is not complete